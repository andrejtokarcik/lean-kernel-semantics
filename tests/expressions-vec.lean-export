1 #NS 0 nat
2 #NS 1 zero
3 #NS 1 succ
4 #NS 0 vector
1 #US 0
1 #EC 2
2 #EC 3
3 #EA 2 1
4 #EA 2 3
5 #EA 2 4
6 #EC 4 1
7 #EC 1
8 #EA 6 7
9 #EA 8 5
