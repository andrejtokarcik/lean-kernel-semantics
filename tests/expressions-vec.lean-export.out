<T>
  <k>
    .
  </k>
  <names>
    0 |-> *empty*
    1 |-> nat
    2 |-> nat zero
    3 |-> nat succ
    4 |-> vector
  </names>
  <universes>
    0 |-> 0
    1 |-> 1
  </universes>
  <expressions>
    1 |-> Const ( nat zero , .List )
    2 |-> Const ( nat succ , .List )
    3 |-> App ( Const ( nat succ , .List ) , Const ( nat zero , .List ) )
    4 |-> App ( Const ( nat succ , .List ) , App ( Const ( nat succ , .List ) , Const ( nat zero , .List ) ) )
    5 |-> App ( Const ( nat succ , .List ) , App ( Const ( nat succ , .List ) , App ( Const ( nat succ , .List ) , Const ( nat zero , .List ) ) ) )
    6 |-> Const ( vector , ListItem ( 1 ) )
    7 |-> Const ( nat , .List )
    8 |-> App ( Const ( vector , ListItem ( 1 ) ) , Const ( nat , .List ) )
    9 |-> App ( App ( Const ( vector , ListItem ( 1 ) ) , Const ( nat , .List ) ) , App ( Const ( nat succ , .List ) , App ( Const ( nat succ , .List ) , App ( Const ( nat succ , .List ) , Const ( nat zero , .List ) ) ) ) )
  </expressions>
  <exit-code>
    0
  </exit-code>
</T>
